assets/js/base/components/label/index.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import elementType from 'prop-types-elementtype';
+import { Fragment } from 'react';
+import classNames from 'classnames';
+
+const Label = ( { label, screenReaderLabel, wrapperElement: Wrapper, wrapperProps } ) => {
+	if ( label && screenReaderLabel && label !== screenReaderLabel ) {
+		return (
+			<Wrapper { ...wrapperProps }>
+				<span aria-hidden>
+					{ label }
+				</span>
+				<span className="screen-reader-text">
+					{ screenReaderLabel }
+				</span>
+			</Wrapper>
+		);
+	}
+
+	if ( ! label && screenReaderLabel ) {
+		if ( typeof Wrapper === 'symbol' && Symbol.keyFor( Wrapper ) === 'react.fragment' ) {
```

Comment:
I'm not sure doing this typeof check for 'symbol' will work in IE11 even with polyfill (should be verified).  Instead you could probably do something like:

```js
if ( label.type && label.type === Fragment )
```

Reply:
Checking the type didn't seem to work, but I refactored the component so we don't need to check for `Symbol` anymore: d398806a803712c6a5d8eb591649c910158b4583. Tested and it works fine on IE11. :+1: